iotedgehubdev/compose_parser.py

```diff
@@ -178,7 +179,8 @@ def service_parser_volumes(create_options_details):
         volumes_list.append(volume_info)
 
     for bind in create_options_details.get('Binds', []):
-        parts = bind.split(':')
+        # Split bind by colons, except those followed by backslashes, which can be a part of Windows paths
+        parts = re.split(r':(?=[^\\])', bind)
```

Comment:
could be c:/kkk ?

Reply:
That is valid, because `volume-name:container-dest` is allowed (`c` is a volume name, `/kkk` is a path inside container): https://docs.docker.com/engine/api/v1.37/#operation/ContainerCreate